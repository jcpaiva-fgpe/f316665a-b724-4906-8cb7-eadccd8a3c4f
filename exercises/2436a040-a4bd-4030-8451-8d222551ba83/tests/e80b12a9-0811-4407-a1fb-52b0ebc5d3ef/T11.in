0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0
0 1 0 0 0 0 0 0 0 0 0 0 0 0 0 0 1 0 0 0
0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0
1 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0
0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0
0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0
0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0
0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0
0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0
0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 1
11
5 4 5 7 2 9
1 1 1 3 10 4
3 9 4 9 5 9
8 2 9 1 10 2
1 8 2 9 4 10
8 2 9 1 10 2
3 1 3 3 2 3
10 10 10 4 8 10
9 1 10 10 10 4
9 2 10 10 4 10
2 7 1 7 1 5
