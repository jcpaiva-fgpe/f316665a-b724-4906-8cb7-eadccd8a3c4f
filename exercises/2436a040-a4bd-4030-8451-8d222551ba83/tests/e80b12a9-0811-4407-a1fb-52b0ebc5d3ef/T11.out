Jogador 1: Agua
Jogador 2: Agua
Jogador 1: Agua
Jogador 2: Agua
Jogador 1: Invalida
Jogador 1: Agua
Jogador 2: Agua
Jogador 1: S1
Jogador 2: Invalida
Jogador 2: Agua
Jogador 1: S1
Venceu o jogador 1
